### ISSUE TITLE
"Company Admin"

### ISSUE THREAD

**Issue by NickolajA** [MicrosoftDocs/SCCMdocs#1702]
Change Company Admin to either Global Administrator or Company Administrator under step 6 in the Note section, to reflect the correct name of the role. Using Company Admin will definitely confuse people as Global Admin is the most commonly used word for this role.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b58dfdf5-6060-e4fa-a806-456ca196ede7
* Version Independent ID: 1f8325fb-518d-412e-f011-7fd62311b087
* Content: [Connect Configuration Manager - Configuration Manager](https://docs.microsoft.com/en-us/sccm/desktop-analytics/connect-configmgr#feedback)
* Content Source: [sccm/desktop-analytics/connect-configmgr.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/desktop-analytics/connect-configmgr.md)
* Product: **configuration-manager**
* Technology: **configmgr-other**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by aczechowski** [comment]
Thanks @NickolajA for the feedback. I did some quick research, and while the terms are somewhat interchangeable (see https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/directory-assign-admin-roles, search for "company admin"), I conferred with some Azure colleagues who confirmed that it's best to generally use "global administrator". There are actually two other instances of the "company admin" term in another article, so I'll take care of them all at the same time.
